plugin.video.livestream/addon.xml

```diff
@@ -1,23 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.livestream" name="Livestream" version="2016.3.29" provider-name="eracknaphobia">
+<addon id="plugin.video.livestream" name="Livestream" version="2017.4.3" provider-name="eracknaphobia">
   <requires>
-   <import addon="xbmc.python" version="2.14.0"/>   
-   <import addon="script.module.simplejson" version="3.3.0"/>    
+   <import addon="xbmc.python" version="2.19.0"/>      
   </requires>
   <extension point="xbmc.python.pluginsource" library="main.py">
         <provides>video</provides>
   </extension>  
   <extension point="xbmc.addon.metadata">    
     <platform>all</platform>
-    <summary lang="en">Livestream.com Videos</summary>
-    <description lang="en">Livestream, formerly known as Mogulus, is a live streaming video platform that allows users to view and broadcast video content using a camera and a computer through the internet.</description>
-    <disclaimer lang="en"></disclaimer>
+    <summary lang="en_GB">Livestream.com Videos</summary>
```

Comment:
According to the wiki: 
any of these elements can be translated into multiple languages and should be added once for each supported language. The lang attribute should contain a locale identifier. If omitted, it defaults to en_GB. (Note: Kodi v14 and older uses ISO-639 code. See List of language codes (ISO-639:1988)).

Since Helix is v14 you'll have to keep it as `en`. Other alternative is to move the addon to a "higher version" repository (setting this one as broken first).